#! /bin/sh /usr/share/dpatch/dpatch-run
## 91_build_pwdecrypt.dpatch by Paul Wise <pabs@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Enable building and installing pwdecrypt (Closes: #472303)

@DPATCH@
diff -urNad nss-3.12.0~beta2~/mozilla/security/nss/cmd/manifest.mn nss-3.12.0~beta2/mozilla/security/nss/cmd/manifest.mn
--- nss-3.12.0~beta2~/mozilla/security/nss/cmd/manifest.mn	2007-01-26 04:04:56.000000000 +0900
+++ nss-3.12.0~beta2/mozilla/security/nss/cmd/manifest.mn	2008-03-27 22:55:06.000000000 +0900
@@ -65,6 +65,7 @@
  pk12util \
  pk11mode \
  pp  \
+ pwdecrypt \
  rsaperf \
  sdrtest \
  selfserv  \
